ast/compile_test.go

```diff
@@ -1734,6 +1734,24 @@ p {
 	}
 }
 
+func TestCompilerCheckSafetyFunctionAndContainsKeyword(t *testing.T) {
```

Comment:
Originally, the check lived under `check.go`, and so the compiler was doing the error checking. I left the extra safety check in, since it's not hurting anything, and it's nice to have a downstream safety check.